Docker/python-nginx/python3.7-alpine/Dockerfile

```diff
@@ -0,0 +1,210 @@
+# Dockerfile written by Sebastian Ramirez <tiangolo@gmail.com> at https://github.com/tiangolo/uwsgi-nginx-docker
```

Comment:
Is all the docker work straight from the source? The `uwsgi-nginx-docker` repo? Couldn't tell how much is a template/boilerplate vs our customizations. Not sure what our typical process is for a base python image

Reply:
at the time i copied most of our `python3.6-alpine3.7` base image from `tiangolo/uwsgi-nginx-docker`, we did make small changes to it overtime